data_diff/database.py

```diff
@@ -261,11 +380,45 @@ def quote(self, s: str):
     def to_string(self, s: str):
         return f"cast({s} as varchar(1024))"
 
+    def select_table_schema(self, path: DbPath) -> str:
+        if len(path) > 1:
+            raise ValueError("Unexpected table path for oracle")
+        (table,) = path
+
+        return (
+            f"SELECT column_name, data_type, 9 as datetime_precision, data_precision as numeric_precision"
```

Comment:
<img width="1020" alt="CleanShot 2022-06-10 at 10 26 19@2x" src="https://user-images.githubusercontent.com/97400/173024288-deabaa8c-1cee-43a4-8a47-b0badd50d372.png">

Default for Oracle seems to be 6, not 9

Reply:
Link please

Reply:
https://docs.oracle.com/cd/B19306_01/server.102/b14225/ch4datetime.htm#i1006081